test/cases/connection_test_sqlserver.rb

```diff
@@ -124,6 +124,31 @@ class ConnectionTestSQLServer < ActiveRecord::TestCase
       end
     end
 
+    should 'auto reconnect when setting is on for stored procedure' do
```

Comment:
Please change these to `it` statements. The mini-should compatibility layer is removed and these will fail in master soon.